edubots_surveys.Rnw

```diff
@@ -560,53 +560,37 @@ opinions.
-questions \textcolor{blue}{are shown in Appendix I (Section \ref{app:questions})}.
+(68.4\%) and 89 at other tertiary education institutions (31.6\%).From those, 179 indicated their gender as male (63.5\%), 98 as female (34.8\%) and 5 teachers preferred not to indicate their gender (1.8\%). In terms of age, the majority of the participants (n=111) in this survey was 45-55 years old (39.4\%), while 91 teachers were 35-45 (32.3\%), 46 were 25-35 (16.3\%) and 34 teachers were older than 55 (12.1\%). Last, 84 teachers had a teaching experience of 16-25 years (29.8\%), 75 teachers 6-15 years of experience (26.6\%), 69 teachers had 0-5 years of experience (24.5\%) and finally, 54 teachers had more than 25 years of teaching experience (19.1\%).
+
+For addressing the 4th research question, a second online survey was administered (i.e., Google Form), in Spanish, comprised by four questions focusing on demographic data (i.e., gender, age, degrees, teaching experience), followed two multiple choice questions with single answers and one multiple choice questions allowing the participants to choose multiple answers. The items of this survey focused on the kind of interaction media features that the teachers value the most. The survey was administered to university teachers teaching at the University of Granada, but also student teachers who attended a teacher formation course (on the use of new technologies in higher education). The responders were reached out through the use of mailing lists, while an invitation to participate in the survey was also promoted in the newsletter of the vice deanship for International Relations of the University of Granada. Survey questions \textcolor{blue}{are shown in Appendix I (Section \ref{app:questions})}. 
+%make this appendix 2 and delete the paragraph before the questions. 
+Responses were stored automatically in a Google Drive spreadsheet. A total of 205 teachers responded to the second survey: 187 graduate teachers (91.2%) and 18 student teachers (8.8%). From those, 124 were (60.5\%), 65 female (31.7\%) and 16 teachers preferred not to indicate their gender (7.8\%). In terms of age, the majority of the participants (n=70) was again, as in the first survey, 45-55 years old (34.1\%), while 67 teachers were 35-45 (32.7\%), 42 were 25-35 (20.5\%) and 26 teachers were older than 55 (12.7\%). In terms of teaching experience, 59 teachers had a teaching experience of 16-25 years (28.8\%), 51 teachers had 6-15 years (24.9\%) and 0-5 years of experience (24.9\%), and 44 teachers more than 25 years of teaching experience (21.5\%). 
 
-Responses were stored automatically in a Google Drive spreadsheet. Eventually,
-the results from the two forms we used for the first survey were collated in a
-single spreadsheet. Next we will show the results and analyse them.
 
 % ********************** RESULTS ****************************
 
 \section{Results and analysis}
 \label{sec:res}
 
-% Describe the survey and population
```

Comment:
So what you have done is to move this from results and analysis to discussion, is that correct?